hoomd/md/integrate.py

```diff
@@ -7,16 +7,17 @@
 # Maintainer: joaander / All Developers are free to add commands for new
 # features
 
+import itertools
+from functools import partial
```

Comment:
Is this import used? It's newly added but I don't see where it it utilized.

Reply:
This is a remnant of an intermediate commit.